avocado/plugins/runner_nrunner.py

```diff
@@ -181,18 +181,19 @@ def _populate_task_logdir(self, base_path, task, statuses, debug=False):
     def _get_all_runtime_tasks(self, test_suite):
         result = []
```

Comment:
I know it is not related to the proposed changes, but it may be a good opportunity to change `result` to `runtime_tasks`.